core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -16,11 +16,11 @@
       <select class="audio-language-select" 
               ng-model="selectedLanguage.value"
               ng-options="o.value as o.displayed for o in languagesInExploration"
-              ng-change="languageSelected()"></select>
+              ng-change="onNewLanguageSelected()"></select>
     </div>
   </div>
-  <img class="audio-toggle-button" ng-if="isAudioBarExpanded" ng-src="<[IMAGE_URL_COLLAPSE_AUDIO_BUTTON]>" ng-click="collapseAudioBar()">
-  <img class="audio-toggle-button" ng-if="!isAudioBarExpanded" ng-src="<[IMAGE_URL_EXPAND_AUDIO_BUTTON]>" ng-click="expandAudioBar()">
+  <div class="audio-collapse-button audio-toggle-button" ng-if="audioBarIsExpanded" ng-click="collapseAudioBar()"><i class="fa fa-sort-up"></i></div>
+  <div class="audio-expand-button audio-toggle-button" ng-if="!audioBarIsExpanded" ng-click="expandAudioBar()">AUDIO <i class="fa fa-sort-down"></i></div>
```

Comment:
Use ng-translate here. (Consider also having the text be normal, e.g. Audio, and use CSS to style it to uppercase -- that might make the base text more in line with the surrounding ones in the translations files.)

Reply:
done